config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryroutines.bigquery.cnrm.cloud.google.com.yaml

```diff
@@ -40,50 +39,88 @@ spec:
       jsonPath: .status.conditions[?(@.type=='Ready')].lastTransitionTime
       name: Status Age
       type: date
-    name: v1beta1
```

Comment:
wait, it looks like this resource is already in Beta?

Reply:
I think this resource is in beta based on https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/bigquery_v1beta1_bigqueryroutine.yaml